app/assets/javascripts/discourse/app/components/user-menu/items-list-item.hbs

```diff
@@ -0,0 +1,25 @@
+<li class={{concat "user-menu-list-item " @item.classNames}}>
```

Comment:
The other advantage of dealing with a single component is that we are able to keep styling consistent in the entire `UserMenu`. Since core and plugin will rely on the same component, it'll allow us to safely restyle this list and links in the future without having to worry that plugins might have relied on the CSS classes here and breaking their styling when we decide to change the CSS classes.